go.sum

```diff
@@ -12,10 +12,6 @@ github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239 h1:kFOfPq6dUM1hTo
 github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239/go.mod h1:2FmKhYUyUczH0OGQWaF5ceTx0UBShxjsH6f8oGKYe2c=
 github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5 h1:0CwZNZbxp69SHPdPJAN/hZIm0C4OItdklCFmMRWYpio=
 github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5/go.mod h1:wHh0iHkYZB8zMSxRWpUBQtwG5a7fFgvEO+odwuTv2gs=
-github.com/asafgabai/gocmd v0.1.20-0.20211104074947-336390e289b0 h1:5DZkniph+OkCKpbBe0TYMMNrgUx4Ml5xNZb3hg/nGPU=
```

Comment:
Reminder for myself to update the go.mod files here and in gocmd and jfrog-client-go according to the expected releases versions.